client/src/partials/sales/sales.js

```diff
@@ -282,7 +282,7 @@ angular.module('bhima.controllers')
       var invoiceRequest = packageInvoiceRequest();
 
       if (!validSaleProperties(invoiceRequest)) { return; }
-      $http.post('sale/', invoiceRequest).then(handleSaleResponse); // FIXME: refactor to using connect
+      $http.post('/sales', invoiceRequest).then(handleSaleResponse); // FIXME: refactor to using connect
```

Comment:
This entire module will essentially be replaced (for example using the integration tested API service instead of `$http`). No problem with keeping comments up to date but this code will not exist.